lib/slack/block_kit/composition/confirmation_dialog.rb

```diff
@@ -15,8 +15,9 @@ def title(text:, emoji: nil)
           self
         end
 
-        def confirm(text:, emoji: nil)
+        def confirm(text:, emoji: nil, style: nil)
           @confirm = PlainText.new(text: text, emoji: emoji)
+          @style = style
```

Comment:
Thanks for this 😄 

Maybe this could be on the top-level `ConfirmationDialog#style`?
Mostly so that it stays on the top-level, rather than nested inside of `confirm`, matching the way the API is
[docs](https://api.slack.com/reference/block-kit/composition-objects#confirm)

Basically something like:
```ruby
def style(value)
  @style = value

  self
end
```

What do you think, @luvtechno?

Reply:
@CGA1123 Yeah, I was wondering which interface is better. The reason I decided to add this option to `#confirm` is that while `style` field exists at the top level, it only affects confirm button, so I thought it is more intuitive for the user of this class to understand what this option means.  

On the other hand, If the user of this class reads the official API doc, it will be more easier to use if `#style` method exists as you suggested. 

I guess both of those approach are valid and coexist? 🤔 

How the design principle of other classes in this gem? Are you making them relatively strict with the official API interface, or rather arranging for usability? I would like to make this consistent with the rest of this gem.

Reply:
> I guess both of those approach are valid and coexist? 🤔

I've added `#style` method while keeping `stlyle` param of `#confirm` method. The format is for consistency with the official API, while the latter for short-hand. 

Please let me know your thought.

Reply:
> How the design principle of other classes in this gem? Are you making them relatively strict with the official API interface, or rather arranging for usability? I would like to make this consistent with the rest of this gem.

A good question. I think it makes sense to follow the official API interface as closely as possible. This example _is_ very minor, but if we go with this path I think it can quickly become messy.

It will also simplify understanding of this gem by not introducing gem-specific edge cases and in the future should make any new or changing features more straight forward.

I think let's remove the `style:` keyword from `#confirm` and go with having `#style` only.

Reply:
Okay! Make sense to me.